pkgs/development/python-modules/django-health-check/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, sphinx
+, setuptools_scm
```

```suggestion
, setuptools-scm
```
setuptools_scm is an alias https://github.com/NixOS/nixpkgs/blob/38b8d261dd599ab976772019e181a2d6097a6e06/pkgs/top-level/python-aliases.nix#L162